neo.UnitTests/SDK/UT_RpcClient.cs

```diff
@@ -0,0 +1,522 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Moq.Protected;
+using Neo.IO;
+using Neo.IO.Json;
+using Neo.Ledger;
+using Neo.Network.P2P.Payloads;
+using Neo.Network.RPC;
+using Neo.Network.RPC.Model;
+using Neo.SmartContract;
+using Neo.SmartContract.Manifest;
+using Neo.VM;
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Neo.UnitTests.SDK
```

Comment:
The `RpcClient` has been moved to `Neo.Network.RPC`. So this namespace should be changed too.

Reply:
Put it the same place with others